pdr_backend/lake/slot.py

```diff
@@ -29,7 +29,6 @@ def __init__(
         self.truevalue = truevalue
         self.roundSumStakesUp = roundSumStakesUp
         self.roundSumStakes = roundSumStakes
```

Comment:
The Slot table is very similar with the Trueval table, we should probably combine this 2 tables just into 1 table called Slot.
We could still read Trueval events from the subgraph in order to update the Slot and other tables with the trueval, but I don't think there is a benefit in storing a Trueval table